docs/Recipes.md

```diff
@@ -56,101 +67,98 @@ And a corresponding (example) `tsconfig.json`
 }
 ```
 
+Please note, that if you want to use synthetic defaults imports, aka things like `import React from 'react'` instead of `import * as React from 'react'` you will need to:
```

Comment:
```suggestion
Please note, that if you want to use synthetic default imports, a.k.a. things like `import React from 'react'` instead of `import * as React from 'react'` you will need to:
```